modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -691,10 +690,6 @@ func (fsrv *FileServer) getEtagFromFile(fileSystem fs.FS, filename string) (stri
 		if err != nil {
 			return "", fmt.Errorf("cannot read etag from file %s: %v", etagFilename, err)
 		}
-
-		// Etags should not contain newline characters
-		etag = bytes.ReplaceAll(etag, []byte("\n"), []byte{})
```

Comment:
This looks unrelated? Was this a mistake in rebase?

Reply:
yep. will fix.